src/Umbraco.Core/Configuration/ClientDependencyConfiguration.cs

```diff
@@ -49,5 +53,57 @@ internal bool IncreaseVersionNumber()
 
             return false;
         }
+
+        /// <summary>
+        /// Clears the temporary files stored for the ClientDependency folder
+        /// </summary>
+        /// <param name="currentHttpContext"></param>
+        internal bool ClearTempFiles(HttpContextBase currentHttpContext)
+        {
+            var cdfTempDirectories = new HashSet<string>();
+            foreach (BaseCompositeFileProcessingProvider provider in ClientDependencySettings.Instance
+                .CompositeFileProcessingProviderCollection)
+            {
+                var path = provider.CompositeFilePath.FullName;
+                cdfTempDirectories.Add(path);
+            }
+
+            try
+            {
+                var relativePath = currentHttpContext.Server.MapPath(XmlFileMapper.FileMapVirtualFolder);

```

Comment:
small thing but this is not the relative path, this is the absolute path of the file